packages/pancake-uikit/src/widgets/Menu/Menu.tsx

```diff
@@ -112,34 +103,22 @@ const Menu: React.FC<NavProps> = ({
   return (
     <Wrapper>
       <StyledNav showMenu={showMenu}>
-        <Logo
-          isPushed={isPushed}
-          togglePush={() => setIsPushed((prevState: boolean) => !prevState)}
-          isDark={isDark}
-          href={homeLink?.href ?? "/"}
-        />
         <Flex>
+          <Logo isDark={isDark} href={homeLink?.href ?? "/"} />
+          {isDesktop && <MenuItems items={links} ml="24px" />}
```

Comment:
I _think_ we also need these displaying > 852px (`md`) - so we'd need something like `isDesktop || isTablet`

Reply:
You're right, I'll fix this.